libs/shared/flagd-js-core/src/lib/feature-flag.ts

```diff
@@ -0,0 +1,202 @@
+import {FlagValue} from '@openfeature/server-sdk';
```

Comment:
We can't use any specific imports from `@openfeature/server-sdk` or `@openfeature/web-sdk`, since this would mean we can't use it in the other runtime.

Hypothetically, all the stuff you're using is actually coming from @openfeature/shared. This is the first use-case we've had of a thing external to the js-sdk that could use that as a dep directly. We could potentially start publishing it to address this...

cc @beeme1mr

Reply:
Yes, that makes sense to me but we should give the package a better name.

Reply:
Makes sense. Proposing packaging name `@openfeature/sdk-common`

Reply:
Will open a new issue discussing this in the js-sdk.

Reply:
Thank you :)

Reply:
Discussed separately at https://github.com/open-feature/js-sdk/issues/616